public/main.js

```diff
@@ -1,31 +1,45 @@
 $(document).on('ready', function(){
 	submitAnswer();
+    createNewMessage();
 });
 
 function submitAnswer(){
 	$('#submit-answer').click(function(){
 		var answerValue = $('#user-answer').val();
-        if(answerValue !== ''){
-            displayAnswer(answerValue);
+        var user = $('#usernick').val();
+        if(answerValue !== '' && user !== ''){
```

Comment:
i have fixed this. you can check and merge, so that i will create new branch and start new task